specification/0001-multifold.instance.json.md

```diff
@@ -0,0 +1,174 @@
+# multifold.instance.json
+
+## Summary
+
+This document acts as a Request for Comments (RPC) on the topic of how a MultiFold instance metadata is formatted.
+
+## Motivation
+
+This RFC describes a scalable, portable, and more powerful format that defines an instance. MultiFold offers users the
+ability install mod loaders and mods seamlessly, without manual configuration.
+
+## Explanation
+
+The following is the schema for instance metadata. The primary format is JSON, however it can be represented in other
```

Comment:
I still believe there is no need for any formats aside from JSON, only further complicating the internals. Not an issue that it's here, but in my opinion should be reconsidered.

Reply:
The point here is that the schema can be used/represented in other formats, the launcher itself is going to read and write in JSON.

Reply:
Perfectly reasonable, all good.